RUN.md

```diff
@@ -162,7 +162,6 @@ services:
     restart: always
     depends_on:
       - proxy
```

Comment:
Hi @tiagojulianoferreira ,

Thank you for your PR. You are totally right: the dependency to acme must be removed. 
But I also think that the `proxy` dependency on the line just above should be renamed to `rproxy` to match the service name associated with traefik. 

Could you please check and make the final change, if applicable?

Reply:
True! I also had a problem with this R